modules/assim.sequential/R/Analysis_sda_multiSite.R

```diff
@@ -293,20 +340,29 @@ GEF.MultiSite<-function(setting, Forecast, Observed, H, extraArg,...){
   y.censored <- as.numeric(ifelse(Y > interval[, 1], Y, 0))
   
   if(t == 1){ #TO DO need to make something that works to pick whether to compile or not
+  # intial Q depends on the size of aqq
+    q.tmp <- diag(1, nrow(aqq), ncol(aqq))
 
+    if (q.type == 1){
```

Comment:
Maybe define q.types with named constants rather than arbitrary integers? Right now it's hard to understand what these two cases represent, or why there isn't a need for the 3rd case